workshops/hcload/README.md

```diff
@@ -0,0 +1,465 @@
+---
+name: Hackclub CDN Uploader
+description: Make a CLI to upload to the Hackclub CDN and implement uploading to the CDN in your apps
+author: '@KhushrajRathod'
+img: https://cloud-bb4wyl0oo.vercel.app/0screenshot_2020-11-11_at_10.00.26_pm.png
+---
+
+# Hackclub CDN Uploader 
+
+<video autoplay loop muted>
+    <source src="https://cloud-7fq4zemw0.vercel.app/0hcloaddemo.mp4" type="video/mp4">
+</video>
+
+Everyone likes unlimited storage, especially when it's available as a cdn (i.e. you can directly embed items uploaded into webpages). If you're on the [Hack Club Slack](https://hackclub.com/slack/), you've probably uploaded something or the other to [#cdn](https://hackclub.slack.com/archives/C016DEDUL87) (If you aren't on the Slack / Haven't uploaded to #cdn, you should check both out!)
+
+Today, we'll be creating a library that works with the Hackclub CDN. Along with that, we'll also create a command-line tool to upload files using our library. We'll be using [Deno](https://deno.land/) to accomplish this. (If you've previously used Node.js, check out [this talk](https://www.youtube.com/watch?v=M3BM9TB-8yA) on Deno)
+
+You should know a bit of Javascript to follow along -- but don't worry, if you don't know JS, you'll just take a bit longer and you'll have to Google somewhat.
+
+Here's the [final code](https://repl.it/@KhushrajRathod/hcload)
+
+If you get stuck anywhere in this workshop, feel free to ask me questions! I'm @KhushrajRathod on the [Hack Club Slack](https://hackclub.com/slack/).
+
+## Part 1: Preparing your environment
+### Preparing repl.it
+
+> Note: Since (at the time of writing) Deno is in Beta @repl.it, you'll want to ignore any errors you see regarding await, import, and the Deno object.
+
+To make it easier for us to code today, we'll be using Repl.it. However, towards the end of the workshop, we'll download [Deno](https://deno.land/) so that we can upload files from our computers.
```

Comment:
I think this should be made a little clearer. You want to communicate that one step of the steps in this workshop will involve a terminal and downloading something to their computer, so if they are using a school computer without access to these things, they shouldn't do this workshop